yelp_package/extra_requirements_yelp.txt

```diff
@@ -32,7 +32,7 @@ srv-configs==1.1.0
 sticht[yelp_internal]==1.1.18
 vault-tools==0.9.2
 yelp-cgeom==1.3.1
-yelp-clog==5.2.3
+yelp-clog==7.1.1
```

Comment:
i think we'll want to add `tenacity` as well since that's now required by yelp-clog

(figuring this out was kinda annoying: i added the direct dependencies (i.e., things we import) into requirements-minimal.txt and copied the contents here to requirements.txt so that i could run `check-requirements`)

Reply:
there's also some lines we can delete: https://fluffy.yelpcorp.com/i/wmSXXqXBvW1THmZPs5mfNGfF2cR9l69g.html is what i think we'd want as a final state

Reply:
So I've pinned deps and ran `check-requirements` for the following main dependencies:
```
yelp-clog
scribereader
clusterman-metrics
vault-tools
slo-transcoder
yelp-profiling
sticht[yelp_internal]==1.1.18
``` 
Not sure about deleting other deps, as I can see most of them being referenced in code

Reply:
oh good catch - i somehow missed grepping for clusterman-metrics (which is maybe why my check-requirements said we could delete some deps)

Reply:
also: big thank you for annotating these dependencies :)